NNNN-Enhance-BodyInformation-vehicle-data.md

```diff
@@ -0,0 +1,140 @@
+# Enhance BodyInformation vehicle data
+
+* Proposal: [SDL-NNNN](NNNN-Enhance-BodyInformation-vehicle-data.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to enhance `BodyInformation` with more params to get access to more door related signals.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. We need to provide apps with doors lock status along with if trunk/liftgate or hood/bonut are ajar. This information is handy for all the apps but in particular the apps which provide safety information and insurance.
```

Comment:
```suggestion
In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. We need to provide apps with doors' lock status along with if trunk/liftgate or hood/bonut are ajar. This information is handy for all the apps but in particular the apps which provide safety information and insurance.
```
I'm not familiar with "bonut" - can you please confirm this is the intended terminology here?

Reply:
I misspelled, It is bonnet. Term used in EU countries to refer to hood.